src/bot/bot.js

```diff
@@ -1,31 +1,40 @@
 const fs = require('fs');
 const path = require('path');
-const { Client, Collection, Intents } = require('discord.js');
-const { botAuth } = require('../const/keys.js')
+const { Client, Options, Intents: { FLAGS } } = require('discord.js');
 const fct = require('../util/fct.js');
-const cronScheduler = require('./cron/scheduler.js');
 const settingModel = require('../models/managerDb/settingModel.js');
 const textModel = require('../models/managerDb/textModel.js');
 const loadCommands = require('./util/cmdLoader');
 
-const flags = Intents.FLAGS
 const intents = [
-    flags.GUILDS,
-    flags.GUILD_MEMBERS, // !!! PRIVILEGED !!!
-    flags.GUILD_BANS,
-    flags.GUILD_EMOJIS_AND_STICKERS,
-    flags.GUILD_INTEGRATIONS,
-    flags.GUILD_WEBHOOKS,
-    flags.GUILD_VOICE_STATES,
-    flags.GUILD_MESSAGES,
-    flags.GUILD_MESSAGE_REACTIONS,
-    flags.GUILD_MESSAGE_TYPING,
-    flags.DIRECT_MESSAGES,
-    flags.DIRECT_MESSAGE_REACTIONS,
-    flags.DIRECT_MESSAGE_TYPING
-]
-
-const client = new Client({ intents: intents });
+  FLAGS.GUILDS,
+  FLAGS.GUILD_MEMBERS,
+  // FLAGS.GUILD_BANS,
+  // FLAGS.GUILD_EMOJIS_AND_STICKERS,
+  FLAGS.GUILD_INTEGRATIONS,
+  // FLAGS.GUILD_WEBHOOKS,
+  FLAGS.GUILD_VOICE_STATES,
+  FLAGS.GUILD_MESSAGES,
+  FLAGS.GUILD_MESSAGE_REACTIONS,
+  // FLAGS.GUILD_MESSAGE_TYPING,
+  // FLAGS.DIRECT_MESSAGES,
```

Comment:
Maybe let direct messages active, simply for the "Please use commands in guild" message? Having no response might look "bot broken".

Reply:
This is only applicable to message commands, and I don't see why we'd need that.